data/building-hours/9-osa.yaml

```diff
@@ -0,0 +1,17 @@
+name: OSA
+subtitle: Office of Student Activities
```

Comment:
This feels like it should be Name and Abbreviation instead of Name and Subtitle...